### ISSUE TITLE
parse_agents crashes RStudio

### ISSUE THREAD

**Issue by assumednormal** [ua-parser/uap-r#8]
I don't have much information beyond this. No errors get returned. No warnings. RStudio just crashes when I try to parse a vector of 1000 user agents. The CLI, however, works.

**Comment by Ironholds** [comment]
Yeah, I've seen this with a couple of packages and I have literally no idea why. It only happens in RStudio and not even all versions of RStudio :/

**Comment by aruneshmathur** [comment]
See this as well. Works from the command line but crashes RStudio. :/

**Comment by Ironholds** [comment]
Aye, I'm trying to rework this. We'll see what we shall see.

**Comment by aruneshmathur** [comment]
Let me know if there's a way I can help. I can paste the RStudio crash report if you like.

**Comment by Ironholds** [comment]
Sure, I'll let you know when I have a beta!
